### DISCUSSION TITLE
[6/21/2023] New Mojo Release!

### DISCUSSION THREAD

**Discussion by goldiegadde** [modular/modular#403] [Announcements]
Hey everyone👋,

We just updated playground with a new Mojo🔥 release ! Checkout the [change log](https://docs.modular.com/mojo/changelog.html) for what's new. 

If you have an active session in the Mojo Playground, you'll need to restart it to get the updated build.

Thanks!

**Comment by anurag629** [comment]
When will you make it open source? I am eagerly waiting for it so that I can use it for my ML projects.

**Comment by lattner** [comment]
We don't have a specific date set, we are working on completing the language:
 https://docs.modular.com/mojo/faq.html#will-mojo-be-open-sourced
